x-pack/plugins/observability/public/components/shared/core_web_vitals/index.tsx

```diff
@@ -100,6 +101,7 @@ export function CoreVitals({
             loading={loading}
             thresholds={CoreVitalsThresholds.FID}
             helpLabel={FID_HELP_LABEL}
+            hasVitals={Boolean(coreVitalPages)}
```

Comment:
used Boolean here, but can use `coreVitalPages > 0` if it's ever expected that this could be negative, which sounds like a data issue to me.

Reply:
Removed `hasVitals` entirely after our conversation regarding the `get_core_web_vitals` endpoint, so no need to worry about this anymore.